app/assets/stylesheets/components/code_listing.css.scss

```
@@ -478,35 +478,44 @@
   .annotation-submission {
     padding: 5px;
 
-    .annotation-controls-block {
-      display: flex;
-      justify-content: space-between;
-      align-items: center;
+    .annotation-submission-input {
+      border-radius: 0;
+      resize: vertical;
     }
 
-    .annotation-submission-button-container {
-      margin-top: 12px;
-      margin-bottom: 12px;
-      text-align: right;
+    .annotation-submission-input.validation-error {
+      border-color: var(--d-danger);
+    }
 
-      .annotation-control-button,
-      .question-control-button {
-        margin-left: 3px;
-      }
+    .saved-annotation-title {
+      position: relative;
+      display: inline-block;
 
-      .annotation-submission-error-list {
-        margin-top: 3px;
-        margin-bottom: 3px;
+      input {
+        padding-left: 40px;
+        width: 220px;
       }
-    }
 
-    .annotation-submission-input {
-      border-radius: 0;
-      resize: vertical;
+      label {
+        color: rgba(var(--d-on-surface-rgb), 0.75);
+        transform: scale(0.85) translateY(-0.4rem);
+        position: absolute;
+        top: 0;
```

```suggestion
        top: 1px;
```

In my Chrome, the label was not aligned with the textbox text.